servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpServerBuilder.java

```diff
@@ -317,46 +267,40 @@ public final HttpServerBuilder appendServiceFilter(final StreamingHttpServiceFil
      * @param factory {@link StreamingHttpServiceFilterFactory} to append.
      * @return {@code this}
      */
-    public final HttpServerBuilder appendServiceFilter(final Predicate<StreamingHttpRequest> predicate,
-                                                       final StreamingHttpServiceFilterFactory factory) {
-        appendServiceFilter(toConditionalServiceFilterFactory(predicate, factory));
-        return this;
-    }
+    HttpServerBuilder appendServiceFilter(Predicate<StreamingHttpRequest> predicate,
+                                          StreamingHttpServiceFilterFactory factory);
```

Comment:
On the `SingleAddressHttpClientBuilder` interface a similar method has `default` implementation. Consider making client and server API consistent and either use `default` in both cases or let the implementation handle it. I don't have a preference.

Reply:
The PR that addresses this suggestion: https://github.com/apple/servicetalk/pull/1866